README.md

```diff
@@ -1 +1,2 @@
-# review-project
\ No newline at end of file
+# review-project
```

Comment:
The purpose of the README.md file is to provide a clear and concise description of the project's purpose and functionality.
I would suggest you provide a brief introduction about your project followed by the technologies, key features, and future features.